src/js/chrome.js

```diff
@@ -28,14 +29,4 @@ libjwt.initPromise.then(() => {
 
 window.insights = window.insights || {};
 
-const init = () => {
-    window.insights.chrome = {
-        ...window.insights.chrome,
-        ...chromeInit(libjwt)
-    };
-};
-
-window.insights = {
-    ...window.insights,
-    ...bootstrap(libjwt, init)
-};
+window.insights = createChromeIntance(libjwt, window.insights);
```

Comment:
should this be `createChromeInstance`?

Reply:
This is basically a constructor which will return the object. Its technically a class but without the unnecessary class boilerplate that JS ads to classes.

Reply:
I just meant the spelling 😄

Reply:
:facepalm:

Reply:
OK its fixed :D sorry about that my eyes don't see that :unicorn: :rainbow: